### ISSUE TITLE
Plug-in failing: invalid API KEY cloudbuild.googleapis.com

### ISSUE THREAD

**Issue by afrietaadmin** [MadaMzandu/uisp-ros-plugin#63]
Please see below

{
  "error": {
    "code": 400,
    "message": "API key not valid. Please pass a valid API key.",
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "API_KEY_INVALID",
        "domain": "googleapis.com",
        "metadata": {
          "service": "cloudbuild.googleapis.com"
        }
      }
    ]
  }
}

**Comment by MadaMzandu** [comment]
the plugin only talks to google for fonts and does not need an api key for this. are you sure this error is from the ros plugin? If so is the user panel loading?

**Comment by afrietaadmin** [comment]
Hi

yes the panel does load.

Is this your Webhook ?

http://localhost/crm/_plugins/ros-plugin/public.php


[image: image.png]

**Comment by MadaMzandu** [comment]
Yes that is the correct webhook. But the format of the error is not from the plugin. The plugin will only issue a code 200 or 202, so the 400 is coming from another entity.  If you are able to enable debugging and see what else it is logging.
